snooty/parser.py

```diff
@@ -159,6 +159,10 @@ def __call__(
         try:
             checksum = self.asset.get_checksum()
             options["checksum"] = checksum
+            if self.asset._width:
+                options["width"] = float(self.asset._width)
```

Comment:
Attributes beginning with an underscore are by convention private. Either they shouldn't have a leading underscore, or `Asset` should have a method that fills out `options` itself. e.g. `options.update(self.asset.get_dimensions_dict())`.

Reply:
done! changed it to accessible attributes with Tuple(float, float)